### PR TITLE
MOEN-32407: Implemented simplified release setup

### PR DESCRIPTION
**Pull request by msoumya-engg-sdk**

### CODE REVIEW

File: MoEngageInApps.podspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Pod::Spec.new do |s|
+  require_relative 'Utils/spec'
+  s.extend MoEngageSDK::Spec
+  s.define
+
+  s.version      = '6.00.3'
+  s.summary      =  "#{s.name} Campaigns for iOS"
+  s.description  = <<-DESC
+                     MoEngageInApp Messaging are custom views which you can send to a segment of users to show custom messages or give new offers or take to some specific pages.
+                     They can be created from your MoEngage dashboard and MOInApp framework enables you to show those InApp campaigns in your iOS Apps.
```
Comment (umangmoe) [inline MoEngageInApps.podspec:11]: MoEngageInApp is the framework name I guess, not MOInApp

**Review by umangmoe — APPROVED** [review]

File: MoEngageInApps.podspec (hunk 1 above)
Comment (msoumya-engg-sdk) [reply MoEngageInApps.podspec]: Updated name

**Review by RakshithaAcharya — CHANGES_REQUESTED** [review]

File: MoEngage-iOS-SDK.podspec [hunk 2]
```diff
@@ -1,30 +1,63 @@
...
+                   This framework acts like a umbrella framework to include all MoEngage frameworks.
+                   Refer individual framework's description for their usages.
                    DESC
 
-  s.homepage     = 'https://www.moengage.com'
-  s.documentation_url = 'https://developers.moengage.com'
-  s.license      = { :type => 'Commercial', :file => 'LICENSE' }
-  s.author       = { 'MobileDev' => 'mobiledevs@moengage.com' }
-  s.social_media_url   = 'https://twitter.com/moengage'
-  s.ios.deployment_target = '11.0'
```
[the remark this answers is not in the document]
Comment (msoumya-engg-sdk) [reply MoEngage-iOS-SDK.podspec:18]: iOS deployment target defined in common place in `Utils/spec` set with `s.define`

File: MoEngageAnalytics.podspec [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  s.description  = <<-DESC
+                     MoEngageAnalytics framework enables you to track required data regarding user and user's activities inside the app to use the same for effective engagement.
+                   DESC
+
+  s.source       = {
+    :http => "https://github.com/moengage/MoEngage-iOS-SDK/releases/download/9.17.5-preview.1/#{s.name}.xcframework.zip",
+    :sha256 => '6ce3de0db2257d9baf1f654e2f59d83ebb4262a0147829ef73ca278bb7e74e97'
+  }   
+
+  s.addDirectUseWarning
```
Comment (RakshithaAcharya) [inline MoEngageAnalytics.podspec:18]: @msoumya-engg-sdk Does this give the default warning default message, if the pod without using any subspecs.

Comment (RakshithaAcharya) [reply MoEngageAnalytics.podspec:18]: @msoumya-engg-sdk I see its defined in utils file. Resolving this

**Review by RakshithaAcharya — APPROVED** [review]

**Pull request closed without merging**